views/pages/admin/contact-us.pug

```diff
@@ -0,0 +1,25 @@
+extends /views/templates/admin-base.pug
+
+append styles
+  link(href="/assets/css/pages/wellness.css" rel="stylesheet")
+  link(href="/assets/css/pages/admin/contact-us.css" rel="stylesheet")
+
+
+block body
+  h1 Contact Us
+
+  h2 Update execs 
+  - const execs = sources.contact.execs
+  iframe(name="redirect-stopper" id="redirect-stopper" style="display: none")
+  form(action="/api/admin/contact-us/execs" method="post" target="redirect-stopper")
+    table(id="execs-list")
```

Comment:
My other comment applies here too, this may need to change in the future to give the website maintainers a little bit more flexibility with the content.

Reply:
^ also referring to the whole file